examples/maps/maps_example_from_dafny.pvl

```diff
@@ -26,7 +26,7 @@ class Maps {
 
     requires (\forall int i; 0 <= i && i < 100; i in keysMap(a) && i in keysMap(b) && a[i] != b[i]);
     void m2(map<int, boolean> a, map<int, boolean> b) {
-        assert (\forall int i; 0 <= i  && i < 100; a[i] || b[i]);
+        assert (\forall int i; 0 <= i  && i < 100; {:a[i]:} || {:b[i]:});
```

Comment:
No, this is not the default behaviour. Indeed the better trigger here would be `{a[i]} {b[i]}` instead of `{a[i], b[i]}` I would say, but that is not the default behaviour. Do you think it should be? We still have the opportunity to change it now :)

Reply:
I can't think of good arguments for the one or the other way, so I guess not.